modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
+				"You can have multiple occurrences of this option.", //
+				"Example: -Pet", //
+				"         -User"));
+		define(parameter().named(TYPE_MAPPINGS).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code types.", //
+				"You can have multiple occurrences of this option.", //
+				"Example: array:List", //
+				"         map:Map"));
+		define(parameter().named(INSTANTIATION_TYPES).optional().withDescription(
+				"Sets mappings between Swagger spec. types and generated code instance types.",
+				"You can have multiple occurrences of this option.", //
+				"Example (in Java): array:ArrayList", //
+				"                   map:HashMap"));
+		define(parameter().named(IMPORT_MAPPINGS).optional().withDescription(
+				"Specifies mappings between a given class and the import that should be used for that class.",
```

Comment:
"Specifies mappings between an unqualified class name and the import that should be used for that class."